Sources/NIO/EventLoop.swift

```diff
@@ -349,15 +350,20 @@ extension TimeAmount: Comparable {
     }
 }
 
-extension TimeAmount {
+extension TimeAmount: AdditiveArithmetic {
+    /// The zero value for `TimeAmount`.
+    public static var zero: TimeAmount {
+        return TimeAmount.nanoseconds(0)
+    }
+    
     public static func + (lhs: TimeAmount, rhs: TimeAmount) -> TimeAmount {
         return TimeAmount(lhs.nanoseconds + rhs.nanoseconds)
     }
-
+    
     public static func - (lhs: TimeAmount, rhs: TimeAmount) -> TimeAmount {
-        return TimeAmount(lhs.nanoseconds - rhs.nanoseconds)
+         TimeAmount(lhs.nanoseconds - rhs.nanoseconds)
```

Comment:
This change won’t compile in Swift 5.0, which we’re still supporting.

Reply:
Fixed. Thank you for the patience.